NEWS.rst

```diff
@@ -94,6 +94,9 @@ Updated ``Bio.Restriction`` to the April 2024 release of REBASE.
 A bug in ``bgzf`` was resolved, restoring the ability to pass a file handle
 directly to ``BgzfWriter``.
 
+A bug in ``Entrez`` was resolved, allowing users to specify the path of their
+local cache.
```

Comment:
How about something like this:

> ``Bio.Entrez.local_cache`` can be set to a directory for caching downloaded DTD/XSD files.

About the same length, and more informative.

Reply:
That works for me! Do I need to make that change and commit again, or is it something we can do from here?

Reply:
You can do it at the command line (edit, commit, push to github), or in the GitHub browser interface if you prefer (just be extra careful not to leave any trailing whitespace).